gpm_core/src/display/list.rs

```diff
@@ -0,0 +1,12 @@
+use console::style;
+use std::borrow::Borrow;
+
+/// Format the input ``list`` as a list of identifier. They aren't escaped.
+///
+/// If ``list`` is empty, return an empty String
+pub fn format_str_id_list<S: Borrow<str>>(list: &[S]) -> String {
+    list.iter()
+        .map(|s| format!("{}", style(s.borrow()).bold()))
+        .collect::<Vec<String>>()
```

Comment:
```suggestion
pub fn format_str_id_list<S: AsRef<str>>(list: &[S]) -> String {
    list.iter()
        .map(|s| format!("{}", style(s.as_ref()).bold()))
        .collect::<Vec<String>>()
```
Usually you want AsRef, not Borrow. The difference between them is small but its usually AsRef one wants.
> If generic code merely needs to work for all types that can provide a reference to related type T, it is often better to use [AsRef<T>] as more types can safely implement it.